lib/miq_automation_engine/engine/miq_ae_engine/drb_remote_invoker.rb

```diff
@@ -110,6 +110,14 @@ class AutomateMethodException < StandardError
   require 'drb'
   require 'yaml'
 
+  module AutomateYamlLoadAliases
+    def safe_load(yaml, aliases: false, **kwargs)
+      super(yaml, aliases: true, **kwargs)
```

Comment:
Very minor, but I think we can use forwarding syntax now

```suggestion
    def safe_load(yaml, aliases: false, ...)
      super(yaml, aliases: true, ...)
```

Reply:
ah yes!

Reply:
Note I'm not 100% this format works - 2.7 implementations this wouldn't work, but I haven't tried with 3.0+

Reply:
sad trombone!  doesn't work.  :-(. Jason thinks it's for forward all and not forward the rest of the args.